src/bt_auto_dumper/_v2/__main__.py

```diff
@@ -53,39 +81,31 @@ def main(apiver: str | None = None):
             new_autovalidator_address=args.set_autovalidator_address,
             new_codename=args.set_codename,
         )
-        print(f"Configuration updated successfully at {config_path}")
+        logger.info(f"Configuration updated successfully at {config_path}")
 
     if not (subnet_identifier := args.subnet_identifier) or not (autovalidator_address := args.autovalidator_address):
         autovalidator_address, subnet_identifier = load_config(config_path=config_path)
 
     wallet = bt.wallet(name="validator", hotkey="validator-hotkey", path="~/.bittensor/wallets")
-    normalized_subnet_identifier = get_normalized_codename_from_server(
-        subnet_identifier, args.subnet_realm, wallet, autovalidator_address
-    )
-    if not normalized_subnet_identifier:
-        raise ValueError("Failed to normalize the codename.")
-    dump_and_upload(normalized_subnet_identifier, args.subnet_realm, wallet, autovalidator_address, args.note)
+    dump_and_upload(subnet_identifier, args.subnet_realm, wallet, autovalidator_address, args.note)
 
 
 def dump_and_upload(
     subnet_identifier: str, subnet_realm: str, wallet: bt.wallet, autovalidator_address: str, note: str
 ):
     """
-    Dump and upload the output of the commands to the AutoValidator
-    Args:
-        subnet_identifier: Subnet Identifier
-        subnet_realm: Subnet Realm
-        wallet: Bittensor wallet object
-        autovalidator_address: AutoValidator Address
-        note: Comment or note for the operation
+    Dump and upload the logs of the commands to the AutoValidator
     Example:
         dump_and_upload("computehorde", "mainnet", "http://localhost:8000", "Test")
     """
 
-    commands = get_commands_from_server(subnet_identifier, subnet_realm, wallet, autovalidator_address)
-
+    # get normalized subnet identifier and commands from the central server
+    subnet_identifier, commands = get_commands_from_server(
```

Comment:
according to docstring and return type of `get_commands_from_server()`, the function only returns commands and does not return a subnet identifier